### PR TITLE
Added baseUrls constants to group all URLs and also __DEV__ to pick f…

### PR DESCRIPTION
**Pull request by adnanjt**
…or production URL

#### Description:

This is a PR to integrate all URLs on baseURLs files and adding __DEV__ variable to choose environments. 

#### How to test:

We need to test all network screens on both develop and production environments.

### CODE REVIEW

File: app/helpers/Fetch.js [hunk 1]
```diff
@@ -1,15 +1,6 @@
...
-    port: 3128,
-    auth: {
-      username: 'covapp',
-      password: 'TIC..2k12*',
-    },
-  },
-  baseURL: 'https://proxyserver.mepyd.gob.do/',
   //url: '/',
   withCredentials: true,
   //httpAgent: new http.Agent({ keepAlive: true }),
```
Comment (inteljudajocu) [inline app/helpers/Fetch.js]: LGTM just remove these comments if they serve no purpose

**Review by inteljudajocu — APPROVED** [review]

File: app/helpers/Fetch.js (hunk 1 above)
Comment (adnanjt) [reply app/helpers/Fetch.js]: thanks Im on it

**Review by JoseFcoEstrella — APPROVED** [review]
LGTM

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -12,10 +12,19 @@ on:
...
+  GOV_DO_TOKEN: ${{ secrets.GOV_DO_TOKEN }}
+
 jobs:
   build-android:
     runs-on: ubuntu-latest
     steps:
+
+      - name: Update .env file
+        run: |
+          printenv >> .env.release
```
Comment (iocedano) [inline .github/workflows/build.yml]:
```suggestion
jobs:
  build-android:
    runs-on: ubuntu-latest
    env:
      MEPYD_C5I_URL: ${{ secrets.MEPYD_C5I_URL }}
      GOV_DO_TOKEN: ${{ secrets.GOV_DO_TOKEN }}
    steps:
      - name: Update .env file
        run: |
          printenv >> .env
```

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -15,7 +15,14 @@ on:
 jobs:
   build-android:
     runs-on: ubuntu-latest
+    env:
+      MEPYD_C5I_URL: ${{ secrets.MEPYD_C5I_URL }}
+      GOV_DO_TOKEN: ${{ secrets.GOV_DO_TOKEN }}
     steps:
+      - name: Update .env file
+        run: |
+          printenv >> .env
```
Comment (iocedano) [inline .github/workflows/build.yml]:
```suggestion
        run: |
          printenv >> .env
          cat .env
```

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -40,7 +40,13 @@ jobs:
         run: echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
 
       - name: Bundle JS
-        run: npx react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res/ --verbose
+        env:
+         MEPYD_C5I_URL: ${{ secrets.MEPYD_C5I_URL }}
+         GOV_DO_TOKEN: ${{ secrets.GOV_DO_TOKEN }}
+        run: |
+          printenv >> .env
+          cat .env
```
Comment (iocedano) [inline .github/workflows/build.yml]:
```suggestion
```

**Comment by iocedano** [comment]
We could use cache action to cache the `.env` files cross steps of workflows
https://help.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows

**Pull request merged**